nncf/experimental/torch/fx/transformations.py

```diff
@@ -37,7 +39,11 @@ def _set_new_node_meta(new_node: torch.fx.Node, prev_node: torch.fx.Node, target
         New node expected to have only one input node.
     :param target_module: Module which is being called by the new node.
     """
-    val = prev_node.meta["val"]
+    val = (
+        prev_node.meta["val"]
+        if prev_node.op not in ["get_attr"]
+        else get_tensor_constant_from_node(prev_node, model).data
+    )
```

Comment:
Is this function tested by a unit test?

Reply:
There is none currently in `test_model_transformer.py`

Reply:
Do we have a model which does ` get_tensor_constant_from_node` in it? If so, after the merge of #2918 it will be tested be a graph test

Reply:
Please copy this test to this PR and expand it in a way to test this if statement https://github.com/openvinotoolkit/nncf/pull/2920/files#diff-2728d5bfe1c37cef7e0a0a22d8b3c127cfcb4ce76a30d704380f1faeb65760afR125-R143